scripts/release-candidate-gradle-plugin.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+function show_banner()
```

Comment:
Can we reuse the banner so it's not duplicated in different files?

Reply:
Good idea, done!!